modules/serverless-rn-1-11-0.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-11-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} 1.11.0
+
+[id="new-features-1-11-0_{context}"]
+== New features
+
+* Knative Eventing on {ServerlessProductName} is now GA.
+* Apache Kafka features such as Kafka channel and Kafka event source are now available as a Technology Preview on {ServerlessProductName}. Kafka integration is delivered through the {ServerlessOperatorName} and does not require a separate community Operator installation. For more information, see the documentation on _Using Apache Kafka with OpenShift Serverless_.
+* {ServerlessProductName} Functions is now delivered as a Developer Preview through the standard Knative `kn` CLI installation. This feature is not yet supported by Red Hat for production deployments, but can be used for development and testing. For more information about using {ServerlessProductName} Functions through the `kn func` CLI, see the link:https://openshift-knative.github.io/docs/docs/functions/about-functions.html[{ServerlessProductName} Functions Developer Preview documentation].
+* {ServerlessProductName} now uses Knative Operator 0.17.0.
```

Comment:
Please remove this line. Otherwise this PR looks good!

Reply:
Cc @cardil